src/main/webapp/404_style.css

```diff
@@ -0,0 +1,41 @@
+@import url(https://fonts.googleapis.com/css?family=Roboto:400,100,300,500);
+
+body {
+	color: #888;
+	font-size: 100%;
+	line-height: 1;
+	font-family: "Roboto", sans-serif;
+}
+
+#main{
+	margin: auto;
+	text-align: center;
+}
+
+p {
+	font-size: 2em;
+	text-align: center;
+	font-weight: 100;
+}
+
+h1 {
+	text-align: center;
+	font-size: 8em;
+	font-weight: 600;
```

Comment:
It looks like we're loading in font weights 400, 100, 300 and 500 on line 1, but here we're using 600. Is this intended?